documentation/src/healthcheck.md

```diff
@@ -0,0 +1,104 @@
+# Healthcheck 🩺
```

Comment:
Love these docs 😍

I don't love the duplication between this and the rustdoc - can they just live in rustdoc?
Maybe link to it from here if you need?